### PR TITLE
Update multiple class and mixin description strings.

### PR DESCRIPTION
**Pull request by djwendo**
Update "description" fields for multiple Classes and Mixins.

### CODE REVIEW

File: schemas/context/segmentdefinition.schema.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "type": "object",
   "meta:extensible": true,
   "meta:abstract": true,
-  "description": "Segment Definition class to describe a segment from any system",
+  "description": "This is a base class for a segment definition which includes required fields like identity of a segment and name of segment. Segment definitions coming from various system can use this class to build Segment Definition XDM schema.",
```
Comment (cfraser) [inline schemas/context/segmentdefinition.schema.json]: `a base class` or `the base class`?

**Comment by amy-durocher** [comment]
These strings are great!  Approved on my side!

File: schemas/context/segmentdefinition.schema.json (hunk 1 above)
Comment (djwendo) [reply schemas/context/segmentdefinition.schema.json]: "a" was provided, but believe "the" would be more accurate as there is no other base class.

**Comment by amy-durocher** [comment]
We are ready to push out changes - no more changes or PRs expected

**Pull request merged**